dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -221,8 +221,8 @@ public WorkflowResource(HttpClient client, SessionFactory sessionFactory, String
         gitHubPrivateKeyFile = configuration.getGitHubAppPrivateKeyFile();
 
         zenodoUrl = configuration.getZenodoUrl();
-        //zenodoClientID = configuration.getZenodoClientID();
-        //zenodoClientSecret = configuration.getZenodoClientSecret();
+        zenodoClientID = configuration.getZenodoClientID();
```

Comment:
Looks like the same as above comment, ditto for imports above

Reply:
Getting refresh token for expired Zenodo and Bitbucket tokens seems to work